pagopa/io-app
# feat: [LLK-49] Add section status banner for unsupported devices

## Short description
This PR adds a local status banner that will be visible to users whose devices do not support a **secure** hardware-based encryption key storage. The banner will inform these users that they will not be able to use future versions of IO on those devices. 

See this [RFC](https://pagopa.atlassian.net/wiki/spaces/IOAPP/pages/642318620/RFC-0003+Banner+in+app+per+notificare+gli+utenti+possessori+di+device+non+supportati) for more insight.

| it | en | 
| - | - |
| <img src="https://user-images.githubusercontent.com/16268789/222101528-22b06767-3a24-4074-8604-38ddab1d2a40.PNG" width="150" /> | <img src="https://user-images.githubusercontent.com/16268789/222101699-cc948966-2cd4-4250-b010-bae6a520a9f1.PNG" width="150" /> |

## List of changes proposed in this pull request
- ts/components/SectionStatus/index.tsx: exports `InnerSectionComponent`
- ts/screens/messages/MessagesHomeScreen.tsx: adds unsupported device banner shown if no key is found.

## How to test
- Verify that the banner is not displayed when Lollipop is disabled, regardless of whether a public key is present or not.
- Confirm that the banner is not displayed when Lollipop is enabled and a public key is present.
- Ensure that the banner is displayed when Lollipop is enabled and a public key is not present.
- Ensure that by tapping on the banner you navigate to the link proposed [here](https://pagopa.atlassian.net/browse/LLK-52).
